test/hotspot/jtreg/compiler/loopopts/superword/SumRed_Long.java

```diff
@@ -25,7 +25,6 @@
  * @test
  * @bug 8076276
  * @summary Add C2 x86 Superword support for scalar sum reduction optimizations : long test
- * @requires vm.bits == "64"
```

Comment:
Maybe we should file a follow-up RFE to check all IR tests that have such a `@requires` limitation and see if we could just change some specific rules to use these new constraints in order to let the test pass for all platforms.